net-mail/checkpassword-pam/checkpassword-pam-0.99-r1.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+IUSE=""
+
+DESCRIPTION="checkpassword-compatible authentication program w/pam support"
+HOMEPAGE="http://checkpasswd-pam.sourceforge.net/"
+SRC_URI="mirror://sourceforge/checkpasswd-pam/${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
Please have a look into the source code :-) License is not correct.

Reply:
fixed.